From f3a945b527afd6f7d1b701e1c64b5de0ed9f7d8b Mon Sep 17 00:00:00 2001
From: Ricky Liang <jcliang@chromium.org>
Date: Sun, 14 May 2017 14:59:57 +0800
Subject: [PATCH] 50-udev-default.rules: set default group for mediaX to video

The upstream patch that covers the change of this patch is:

https://github.com/systemd/systemd/commit/9bfc0df113edd50bd01acba02a42f2a13db83d71
---
 rules/50-udev-default.rules | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rules/50-udev-default.rules b/rules/50-udev-default.rules
index e9eeb85..72c1ff3 100644
--- a/rules/50-udev-default.rules
+++ b/rules/50-udev-default.rules
@@ -34,6 +34,7 @@ SUBSYSTEM=="video4linux", GROUP="video"
 SUBSYSTEM=="graphics", GROUP="video"
 SUBSYSTEM=="drm", GROUP="video"
 SUBSYSTEM=="dvb", GROUP="video"
+SUBSYSTEM=="media", GROUP="video"
 
 SUBSYSTEM=="sound", GROUP="audio", \
   OPTIONS+="static_node=snd/seq", OPTIONS+="static_node=snd/timer"
-- 
2.12.2

